src/main/resources/view/DarkTheme.css

```diff
@@ -343,10 +343,103 @@
 }
 
 #tags .label {
-    -fx-text-fill: white;
-    -fx-background-color: #3e7b91;
     -fx-padding: 1 3 1 3;
     -fx-border-radius: 2;
     -fx-background-radius: 2;
     -fx-font-size: 11;
 }
+
+#tags .red {
+    -fx-text-fill: white;
+    -fx-background-color: #f44336;
+}
+
+#tags .pink {
+    -fx-text-fill: white;
+    -fx-background-color: #e91e63;
+}
+
+#tags .purple {
+    -fx-text-fill: white;
+    -fx-background-color: #9c27b0;
+}
+
+#tags .deepPurple {
+    -fx-text-fill: white;
+    -fx-background-color: #673ab7;
+}
+
+#tags .indigo {
+    -fx-text-fill: white;
+    -fx-background-color: #3f51b5;
+}
+
+#tags .blue {
+    -fx-text-fill: white;
+    -fx-background-color: #1e88e5;
```

Comment:
`#1e88e5` has a HSB lightness of 50.8%, maybe black text is slightly better?

Reply:
Hmm I think either black or white looks fine to the eyes. For consistency I'm following the [2014 Material Design color palettes](https://material.io/design/color/the-color-system.html#tools-for-picking-colors).